sklearn/metrics/_scorer.py

```diff
@@ -413,9 +457,24 @@ def get_scorer(scoring):
     return scorer
 
 
-def _passthrough_scorer(estimator, *args, **kwargs):
-    """Function that wraps estimator.score"""
-    return estimator.score(*args, **kwargs)
+class _passthrough_scorer(_BaseScorer):
+    def __init__(self, estimator):
+        self._kwargs = {}
+        self._score_func = estimator.score
```

Comment:
this is the problem. Is it needed? Can we use a property instead?

Reply:
Indeed they're not needed I think.